Library/Homebrew/service.rb

```diff
@@ -112,6 +112,18 @@ def keep_alive(value = nil)
       end
     end
 
+    sig { params(value: T.nilable(T::Boolean)).returns(T.nilable(T::Boolean)) }
+    def launch_only_once(value = nil)
+      case T.unsafe(value)
+      when nil
+        @launch_only_once
+      when true, false
+        @launch_only_once = value
+      else
+        raise TypeError, "Service#launch_only_once expects a Boolean"
+      end
+    end
```

Comment:
If you'd like me to write tests to cover the nil and TypeError case of this let me know. Other config values don't have tests for these, so I don't see it being necessary, but always open to writing more tests :slightly_smiling_face:

Reply:
I won't stop you from writing more tests. The main problem was that I hadn't found a good way to do that yet.